cfme/tests/intelligence/reports/test_crud.py

```diff
@@ -14,39 +14,39 @@
 from cfme.utils.wait import wait_for_decorator
 
 
-report_crud_dir = data_path.join("reports_crud")
-schedules_crud_dir = data_path.join("schedules_crud")
+REPORT_CRUD_DIR = data_path.join("reports_crud")
+SCHEDULES_CRUD_DIR = data_path.join("schedules_crud")
 
 
 def crud_files_reports():
     result = []
-    if not report_crud_dir.exists:
-        report_crud_dir.mkdir()
-    for file_name in report_crud_dir.listdir():
+    if not REPORT_CRUD_DIR.exists:
+        REPORT_CRUD_DIR.mkdir()
```

Comment:
Can't you return the empty list early at this point, since if the directory doesn't exist then there won't be any file names to parametrize on.